csharp-basics/exercises/Collections/FlightPlanner/Program.cs

```diff
@@ -2,22 +2,95 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace FlightPlanner
 {
     class Program
     {
-        private const string Path = "../FlightPlanner/flights.txt";
+        private const string Path = "../../flights.txt";
 
         private static void Main(string[] args)
         {
             var readText = File.ReadAllLines(Path);
-            foreach (var s in readText)
+            var userInput = "";
+
+            do
             {
-                Console.WriteLine(s);
-            }
+                Console.WriteLine("What would you like to do: \n" +
+                                  "To display list of the cities press 1\n" +
+                                  "To exit program press #");
+                userInput = Console.ReadLine();
+
+                if (userInput == "1")
+                {
+                    Console.WriteLine("To select a city from which you would like to start press 1");
+                    var userInputCity = Console.ReadLine();
+
+                    if (userInputCity == "1")
+                    {
+                        var fromCityList = new List<string>();
+
+                        foreach (var line in readText)
+                        {
+                            var allCities = line.Split('-', '>');
+                            fromCityList.Add(allCities[0].Trim());
+                        }
+
+                        Console.Write(string.Join(", ", fromCityList.Distinct()) + "\n");
+
+                        Console.Write("Enter starting city: ");
+                        var start = Console.ReadLine();
+
+                        foreach (var line in readText)
+                        {
+                            var way = line.Split('-', '>');
+                            var fromCity = way[0].Trim();
+                            var toCity = way[way.Length - 1].Trim();
+
+                            if (fromCity == start)
+                            {
+                                Console.WriteLine(toCity);
+                            }
+                        }
+
+                        var resultCityList = new List<string> { start };
+
+                        var equalToStart = false;
```

Comment:
Garas metodes vajag dalīt mazākās metodēs

Reply:
domā to equalToStart savādāk, vai tam resultCitList nākamajā rindā likt .add(start)?

Reply:
Vispār garas metodes nevajag rakstīt. Ko var, to ieliek mazākās metodēs un galvenajā izcauc tad apakšmetodi.